### PR TITLE
Update and fixes to spec

### PR DESCRIPTION
**Pull request by ttoivone**
Update also automatically generated index.html.

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -281,29 +268,45 @@ image.
 
 Issue: How to express encoded size vs. visible size vs. natural size <a href="https://github.com/WICG/web-codecs/issues/26">WICG#26</a>
 
-### Create a VideoFrame ### {#video-frame-constructor}
+### Create VideoFrame ### {#video-frame-constructor}
 <div algorithm="to create a video frame">
-    : input
-    :: |init|, a [=dictionary=] object of type {{VideoFrameInit}}
-    :: |source|, a {{ImageBitmap}} object.
-    : output
```
Comment (riju) [inline index.bs]: Let's keep this format ?

**Review by riju — APPROVED** [review]
Thanks @ttoivone , overall looks good, just a small nit with the :input :output format for the algo steps.

File: index.bs (hunk 1 above)
Comment (ttoivone) [reply index.bs]: Ok. I'll update the PR

Comment (ttoivone) [reply index.bs]: Updated.

**Pull request merged**